operators/cmd/main.go

```diff
@@ -17,12 +17,18 @@ var log = logf.Log.WithName("main")
 func main() {
 	var rootCmd = &cobra.Command{Use: "elastic-operator"}
 	rootCmd.AddCommand(manager.Cmd)
-
+	logLevelFlag := "log-level"
 	// development mode is only available as a command line flag to avoid accidentally enabling it
 	rootCmd.PersistentFlags().BoolVar(&dev.Enabled, "development", false, "turns on development mode")
+	rootCmd.PersistentFlags().String(logLevelFlag, "INFO", "sets the log level. May be either INFO or DEBUG")
 
 	cobra.OnInitialize(func() {
-		logf.SetLogger(logf.ZapLogger(dev.Enabled))
+		level, _ := rootCmd.Flags().GetString(logLevelFlag)
+		logLevel := false
```

Comment:
Nit: I would tend to name the `logLevel` boolean variable `enableDebugLevel` instead of `logLevel` to better reflect what it does and then name `level` in `logLevel`.

Reply:
I thought about that but wasn't sure if we might want to have a separate debug mode (maybe enabling profiling if we're not already?). That's not a very persuasive argument though so I'm good with changing it